_automating-workflows/api/provision-workflow.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Provision a workflow
+parent: Workflow API
+nav_order: 30
+---
+
+# Provision a workflow
+
+Provisioning a workflow refers to a one-time setup process, usually performed by a cluster administrator to create resources which will be used by end users.  
+
+The `workflows` field of the template may contain multiple workflows. The workflow with the key `provision` can be executed with this API. This API is also executed when the [Create or Update Workflow API]({{site.url}}{{site.baseurl}}/automating-workflows/api/create-workflow/) is called with the `provision` parameter set to `true`.
+
```

Comment:
I am adding code now to prevent provisioning a workflow that has already been provisioned.  This is very similar (and will use the same code) as the create workflow not allowing you to update after it's been provisioned:
```
You can only update a workflow if it has not yet been provisioned.
{: .note}
```
Can we add a similar note:
```
You can only provision a workflow if it has not yet been provisioned. Deprovision the workflow if you need to repeat provisioning.
{: .note}
```